chat-plugins/info.js

```diff
@@ -1699,28 +1703,28 @@ exports.commands = {
 			// Special case for Meowstic-M
 			if (speciesid === 'meowstic') speciesid = 'meowsticm';
 			if (pokemon.tier === 'CAP') {
-				this.sendReplyBox("<a href=\"http://www.smogon.com/cap/pokemon/strategies/" + speciesid + "\">" + generation.toUpperCase() + " " + Chat.escapeHTML(formatName) + " " + pokemon.name + " analysis preview</a>, brought to you by <a href=\"http://www.smogon.com\">Smogon University</a> <a href=\"https://smogon.com/cap/\">CAP Project</a>");
+				this.sendReplyBox(`<a href="http://www.smogon.com/cap/pokemon/strategies/${speciesid}">${generation.toUpperCase()} ${Chat.escapeHTML(formatName)} ${pokemon.name} analysis preview</a>, brought to you by <a href="http://www.smogon.com">Smogon University</a> <a href="http://smogon.com/cap/">CAP Project</a>`);
```

Comment:
Shouldn't this be using `Chat.html` instead of `Chat.escapeHTML`?

Reply:
you commented on something extremely similar above a few months ago, nobody has made a commit yet so that's why it hasn't changed.

Reply:
Sorry about that, I hadn't spotted my earlier comment further down the page. (The reason I came back to this was because I noticed it after the `https` change.)